docs/settings/reporting-settings.asciidoc

```diff
@@ -7,71 +7,108 @@
 * <<reporting-kibana-server-settings,Control how the {report-features} communicate with the {kib} server>>
 * <<reporting-job-queue-settings,Manage background jobs>>
 * <<reporting-capture-settings,Capture screenshots>>
+* <<reporting-network-policy,Restrict requests with a Reporting network policy>>
 * <<reporting-csv-settings,Increase the byte limit for CSV exports>>
 
 [float]
 [[general-reporting-settings]]
-==== General reporting settings
+==== Enable reporting
 
-[cols="2*<"]
-|===
-| [[xpack-enable-reporting]]`xpack.reporting.enabled` {ess-icon}
-  | Set to `false` to disable the {report-features}.
+[[xpack-enable-reporting]]`xpack.reporting.enabled` {ess-icon}::
+When `true`, enables the {report-features}. The {report-features} are automatically enabled in {kib}.
 
-|[[xpack-reporting-encryptionKey]] `xpack.reporting.encryptionKey` {ess-icon}
-  | Set to an alphanumeric, at least 32 characters long text string. By default, {kib} will generate a random key when it
-  starts, which will cause pending reports to fail after restart. Configure this
-  setting to preserve the same key across multiple restarts and multiple instances of {kib}.
+[float]
+[[encryption-keys]]
+==== Encryption key setting
+
+By default, an encryption key is generated for the {report-features} each
+time you start {kib}. If a static encryption key is not persisted in
+the {kib} configuration, any pending reports fail when you restart {kib}.
+
+If you are load balancing across multiple {kib} instances, each instance needs to have
+the same reporting encryption key. Otherwise, report generation fails if a
+report is queued through one instance, and another instance picks up the job
+from the report queue. The instance that picks up the job is unable to decrypt the
+reporting job metadata.
+
+[[xpack-reporting-encryptionKey]] `xpack.reporting.encryptionKey` {ess-icon}::
+The static encryption key for reporting. Use an alphanumeric text string that is at least 32 characters. By default, {kib} generates a random key when it starts, which causes pending reports to fail after restart. Configure `xpack.reporting.encryptionKey` to preserve the same key across multiple restarts and multiple {kib} instances.
+
+[source,yaml]
+--------------------------------------------------------------------------------
+xpack.reporting.encryptionKey: "something_secret"
+--------------------------------------------------------------------------------
+
+[float]
+[[report-indices]]
+==== Reporting index setting
 
-|===
+If you divide workspaces in an Elastic cluster using multiple {kib} instances with a different `kibana.index` setting per instance, you must set a unique `xpack.reporting.index` setting per `kibana.index`. Otherwise, report generation periodically fails if a report is queued through an instance with one `kibana.index` setting, and an instance with a different `kibana.index` attempts to claim the job.
+
+`xpack.reporting.index`::
```

Comment:
This paragraph seems like it should be switched with the previous one, which starts on line 52.